code/src/pages/MovieDetail.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from 'react'
+import { useParams, Link } from 'react-router-dom'
+
+import arrow from "../components/back-arrow.png"
+
+export const MovieDetail = () => {
+  const { id } = useParams()
+  const [movieDetail, setMovieDetail] = useState ([])
+  const MOVIE_DETAIL_URL = `https://api.themoviedb.org/3/movie/${id}?api_key=2c2891737dd5ffa64a8858cdba25ddfe&language=en-US`
+
+  useEffect(() => {
+    fetch(MOVIE_DETAIL_URL)
+    .then((res) => res.json())
+    .then((json) => {
+      setMovieDetail(json)
+      console.log(json) 
+    })
+  }, [MOVIE_DETAIL_URL])
+
+  return (
+    <div> {movieDetail.length===0 ?<></> :    
+      <section className="movie-details-page" style={{backgroundImage:`url(https://image.tmdb.org/t/p/w1280/${movieDetail.backdrop_path})`}}>
+        <Link to={`/`} style={{ textDecoration: 'none' }}>
+          <div className="back-arrow-wrapper">
```

Comment:
Although we saw in the Technigo brief they use inline styling, they have also mentioned it is not the best practice. Maybe instead use a css for this details. Or maybe Karol could enlighten us of When this is suppose to be used 🤠 🤠